flashinfer/gemm/kernels/cutile/gemm_alpha_beta_cutile.py

```
@@ -0,0 +1,490 @@
+                        EPILOGUE_SUBTILE=0,
+                    )
+
+
+def _get_default_kernel_configs():
+    """
+    Get GPU-specific default kernel configs for non-autotune path.
+    """
+    gpu_capability = torch.cuda.get_device_capability()
+
+    if gpu_capability == (10, 0):
+        # Blackwell SM100 – aggressive config with epilogue subtiling
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)